include/xprec/ddouble.h

```diff
@@ -297,6 +298,7 @@ bool isinf(DDouble x);
 bool isnan(DDouble x);
 bool isnormal(DDouble x);
 bool iszero(DDouble x);
+std::string to_string(DDouble d, size_t nDigits = 34);
```

Comment:
This is extending the `to_string` interface. I am not sure that is what we want to do. I think it is better to respect to use `<<` (which can then be used together with a stringstring).

Reply:
You're right.